bedrock/firefox/tests/test_base.py

```diff
@@ -499,6 +508,60 @@ def test_fx_93_0_0_locale(self, render_mock):
         template = render_mock.call_args[0][1]
         assert template == ["firefox/whatsnew/index-account.html"]
 
+    @override_settings(DEV=False)
+    def test_fx_93_0_0_china_en(self, render_mock):
+        """Should use standard whatsnew template for 93.0 in China (en-US)"""
+        req = self.rf.get("/firefox/whatsnew/", HTTP_CF_IPCOUNTRY="cn")
+        req.locale = "en-US"
+        self.view(req, version="93.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/index-account.html"]
+
+    @override_settings(DEV=False)
+    def test_fx_93_0_0_china_de(self, render_mock):
+        """Should use standard whatsnew template for 93.0 in China (de)"""
+        req = self.rf.get("/firefox/whatsnew/", HTTP_CF_IPCOUNTRY="cn")
+        req.locale = "de"
+        self.view(req, version="93.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/index-account.html"]
+
+    @override_settings(DEV=False)
+    def test_fx_93_0_0_china_fr(self, render_mock):
+        """Should use standard whatsnew template for 93.0 in China (fr)"""
+        req = self.rf.get("/firefox/whatsnew/", HTTP_CF_IPCOUNTRY="cn")
+        req.locale = "fr"
+        self.view(req, version="93.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/index-account.html"]
+
+    @override_settings(DEV=False)
+    def test_fx_93_0_0_china_it(self, render_mock):
+        """Should use standard whatsnew template for 93.0 in China (it)"""
+        req = self.rf.get("/firefox/whatsnew/", HTTP_CF_IPCOUNTRY="cn")
+        req.locale = "it"
+        self.view(req, version="93.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/index-account.html"]
+
+    @override_settings(DEV=False)
+    def test_fx_93_0_0_china_es(self, render_mock):
+        """Should use standard whatsnew template for 93.0 in China (es-ES)"""
+        req = self.rf.get("/firefox/whatsnew/", HTTP_CF_IPCOUNTRY="cn")
+        req.locale = "es-ES"
+        self.view(req, version="93.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/index-account.html"]
+
+    @override_settings(DEV=False)
+    def test_fx_93_0_0_china_nl(self, render_mock):
+        """Should use standard whatsnew template for 93.0 in China (nl)"""
+        req = self.rf.get("/firefox/whatsnew/", HTTP_CF_IPCOUNTRY="cn")
+        req.locale = "nl"
+        self.view(req, version="93.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/index-account.html"]
+
```

Comment:
These are 100% fine by me as separate tests, but if you wanted to reuse them more briefly, the `@pytest.mark.parametrize` decorator is very handy - see https://docs.pytest.org/en/6.2.x/parametrize.html or grep for where we already use it in some places

Reply:
eg something like

```
@pytest.mark.parametrize(
    'locale',  # this gets passed in as a new param to the test func
    (
        "en-US",
        "de",
        "fr",
        "it",
        "es-ES",
        "nl",
    ),
    # ids is an optional list of titles/labels for the test params above
    ids=(
        "Should use standard whatsnew template for 93.0 in China (en-US)",
        "Should use standard whatsnew template for 93.0 in China (de)",
        "Should use standard whatsnew template for 93.0 in China (fr)",
        "Should use standard whatsnew template for 93.0 in China (it)",
        "Should use standard whatsnew template for 93.0 in China (es-ES)",
        "Should use standard whatsnew template for 93.0 in China (nl)",
    )
)
@override_settings(DEV=False)
def test_fx_93_0_0_china__standard_template(self, render_mock, locale):
    """Should use standard whatsnew template for 93.0 in China (en-US)"""
    req = self.rf.get("/firefox/whatsnew/", HTTP_CF_IPCOUNTRY="cn")
    req.locale = locale
    self.view(req, version="93.0")
    template = render_mock.call_args[0][1]
    assert template == ["firefox/whatsnew/index-account.html"]
```

Reply:
Thanks! Yeah I'm sure this could be more succinct. One of the things we've kind of got into the habbit of doing for /whatsnew pages is making things really easy to rip out again, as there's a lot of churn for these pages and they only last we few months or so. As such, I didn't spend a great deal of time on refactoring things :)

Reply:
That makes good sense 👍